apps/fishing-map-e2e/src/e2e/fishing-activity/vms.cy.ts

```diff
@@ -0,0 +1,23 @@
+import { getVmsActivityLayerPanel } from '../../support/app.po'
+
+describe('vms', () => {
+  beforeEach(() => cy.visit('/'))
+
+  it('should include all sources available publicly', () => {
+    // Open the filter popup
+    getVmsActivityLayerPanel().find(`[data-test*="activity-layer-panel-btn-filter-vms-"]`).click()
+
+    // Check that all vms layers are available
+    cy.getBySelLike('activity-filters-option').contains('AIS')
```

Comment:
I don't know if this is a good practice, but should we use [this list](https://github.com/GlobalFishingWatch/frontend/blob/master/apps/fishing-map/data/default-workspaces/workspace.production.ts#L64) of datasets instead of maintaining a fixed list here?